### PR TITLE
DM-33836: add enums.ATWhiteLight

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

**Review by patrickingraham — COMMENTED** [review]
Nice. See comments.
Again, I did not look (much) at the code itself.

File: python/lsst/ts/idl/enums/ATWhiteLight.py [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    UNKNOWN = -1
+    AUTOSTART = 0
+    STANDBY = 1
+    RUN = 2
+    SAFETY = 3
+    TEST = 4
+
+
+class ChillerL1Alarms(enum.IntFlag):
+    """Level 1 chiller alarms."""
```
[the remark this answers is not in the document]
Comment (r-owen) [reply python/lsst/ts/idl/enums/ATWhiteLight.py:47]: I haven't a clue. I'd need proper Chiller documentation to understand that.

File: python/lsst/ts/idl/enums/ATWhiteLight.py [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+class ChillerControllerState(enum.IntEnum):
+    """State of the chiller controller"""
+
+    UNKNOWN = -1
+    AUTOSTART = 0
+    STANDBY = 1
+    RUN = 2
+    SAFETY = 3
+    TEST = 4
```
[the remark this answers is not in the document]
Comment (r-owen) [reply python/lsst/ts/idl/enums/ATWhiteLight.py:43]: No. This is the "chiller state" reported as the first byte returned by the "read watchdog" command. The chiller TCP/IP interface is part of the new documentation and it only lists those states:

1 | CS | Control Status Mode |   | 0-4 | 0 - Auto-start, 1 - Standby, 2 - Run, 3 - Safety, 4 - Test

File: python/lsst/ts/idl/enums/ATWhiteLight.py [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    REAR_LEFT_FAN_OPEN = 0x1000_0000
+    REAR_RIGHT_FAN_OPEN = 0x2000_0000
+    FRONT_LEFT_FAN_OPEN = 0x4000_0000
+    FRONT_RIGHT_FAN_OPEN = 0x8000_0000
+
+
+class ChillerWarnings(enum.IntFlag):
+    """Chiller warnings."""
+
+    LOW_PROCESS_FLOW = 0x01
```
[the remark this answers is not in the document]
Comment (r-owen) [reply python/lsst/ts/idl/enums/ATWhiteLight.py:166]: I tried to use chiller terminology. I don't have a manual so I have no idea what any of these warnings or alarms do. I don't even know which fan is which (the TCP/IP interface uses "left/right front/back" to refer to the fans in one place and "1, 2, 3, 4" in another, and I have no idea which is which!

File: python/lsst/ts/idl/enums/ATWhiteLight.py [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+    FAN_FAILURE = 0x8_0000
+
+    CURRENT_SENSOR_2 = 0x10_0000
+    INTERNAL_2_5V_REFERENCE = 0x20_0000
+    INTERNAL_5V_REFERENCE = 0x40_0000
+    SYSTEM_ERROR = 0x80_0000
+
+
+class ChillerL21Alarms(enum.IntFlag):
+    """Level 2 set 1 chiller alarms."""
```
[the remark this answers is not in the document]
Comment (r-owen) [reply python/lsst/ts/idl/enums/ATWhiteLight.py:81]: Level two is broken into two subsections: set 1 and set 2. Beyond that I haven't a clue.

File: doc/version_history.rst [hunk 5]
```diff
@@ -6,6 +6,11 @@
 Version History
 ===============
 
+v?
```
[the remark this answers is not in the document]
Comment (r-owen) [reply doc/version_history.rst]: It turns out I'm the only one messing about in ts_idl so I have added a proper version. i'll be tagging it as soon as this PR is approved and I can merge it.

File: python/lsst/ts/idl/enums/ATWhiteLight.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (r-owen) [reply python/lsst/ts/idl/enums/ATWhiteLight.py]: Note: I agree that "process flow" is opaque so I all instances of "process flow" to "coolant flow" or "coolant flow rate", as seemed most appropriate, here (PROCESS->COOLANT) and in ts_atwhitelight, ts_xml, and ts_config_atcalsys.

File: python/lsst/ts/idl/enums/ATWhiteLight.py [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+    "LampBasicState",
+    "LampControllerError",
+    "LampControllerState",
+]
+
+import enum
+
+
+class ChillerControllerState(enum.IntEnum):
+    """State of the chiller controller"""
```
Comment (wvreeven) [inline python/lsst/ts/idl/enums/ATWhiteLight.py:36]: Nitpicking but please end the sentence with a `.`.

**Review by wvreeven — APPROVED** [review]
Looks good to me. Please review my only comment before merging.

**Pull request merged**